scripts/helpers/utils.py

```diff
@@ -22,19 +26,26 @@ def import_lora_lycoris():
         lora = import_module("lora")
         extra_networks_lora = import_module("extra_networks_lora")
     except ModuleNotFoundError:
-        print("The Lora extension is not installed in the \"extensions-builtin\" file path, cannot overwrite")
+        out("The Lora extension is not installed in the \"extensions-builtin\" file path, cannot overwrite")
 
     try:
         lycoris = import_module("lycoris")
     except ModuleNotFoundError:
-        print("The LyCORIS extension is not installed in the \"extensions-builtin\" file path, cannot overwrite")
+        out("The LyCORIS extension is not installed in the \"extensions-builtin\" file path, cannot overwrite")
 
     return lora, extra_networks_lora, lycoris
 
 
 lora, extra_networks_lora, lycoris = import_lora_lycoris()
 
 
+def make_network_path(network_type):
```

Comment:
Does not seem neccessary seeing as we can use A1111 cmd-args - 4ec8b9e

Reply:
sounds good